.gitignore

```diff
@@ -3,8 +3,12 @@ Cargo.lock
 *.so
 si-units/docs/_build
 si-units/docs/generated
-/venv
 .ipynb_checkpoints
 *.ipynb
 __pycache__
-test_readme.py
\ No newline at end of file
+test_readme.py
+*.lock
+
+# venv
+/venv
+.venv
```

Comment:
add the directory created my mkdoc here?

Reply:
Good idea! I added the `site` directory (amongst others that are built during packaging).